docs/sources/flow/reference/stdlib/nonsensitive.md

```diff
@@ -0,0 +1,28 @@
+---
+aliases:
+- ../../configuration-language/standard-library/nonsensitive/
+title: nonsensitive
+---
+
+# nonsensitive
+
+`nonsensitive` converts a [secret][] value back into a string.
+
+> **WARNING**: Only use `nonsensitive` when you are positive that the value
+> being converted back to a string is not a sensitive value.
+>
+> Strings resulting from calls to `nonsensitive` will be shown in plaintext in
+> the UI.
```

Comment:
`and API`? Or maybe a broader term?

Reply:
I put "and HTTP calls to the UI," since the API the UI uses is undocumented and essentially internal with no stability guarantees.